pegasus/sites.v3/code.org/views/mobile_footer_responsive.haml

```diff
@@ -1,11 +1,48 @@
--# an utra-simple mobile footer
+%link{:href=>'/css/mobile-footer.css', :rel=>'stylesheet'}
+%script{src: '/js/mobile-footer.js'}
 
 %footer#pagefooter
-  .content{:style=>'width:100%'}
-    .left{:style=>'float:left'}
-      - if request.language == "en"
-        %a.whitefooterlink{:href=>"/privacy", :style=>"color: #ffa400;"} Privacy Policy
-      %br
-      !="&copy; Code.org, #{Time.now.year}"
-    .right{:style=>'float:right'}
```

Comment:
So I think this is the section we need to keep for non-EN. What I would do:

```
- if request.language != "en"
  .content{:style=>'width:100%'}
    .left{:style=>'float:left'}
      !="&copy; Code.org, #{Time.now.year}"
    .right{:style=>'float:right'}
      =view :language
- else
  #footer-links
    ...
```

The way I think about it is that we have a full-size footer and a mobile footer. In each view we determine what that looks like for EN and non-EN. So a 2x2 matrix.